documentation/book/con-maintenance-time-windows-overview.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// assembly-maintenance-time-windows.adoc
+
+[id='con-maintenance-time-windows-overview-{context}']
+= Maintenance time windows overview
+
+In most cases, the Cluster Operator only updates your Kafka or Zookeeper clusters in response to changes to the corresponding `Kafka` resource.
+This enables you to plan when to plan when to apply changes to a `Kafka` resource to minimize the impact on Kafka client applications.
```

Comment:
Spotted a typo:

This enables you to plan when to plan when to apply changes... > This enables you to plan when to apply changes...